fuzzers/binary_only/qemu_launcher/src/instance.rs

```diff
@@ -188,12 +188,14 @@ where
             |_, _, _, _| Ok(self.options.tui),
             tuple_list!(AflStatsStage::builder()
                 .map_observer(&edges_observer)
+                .map_name("map_feedback")
                 .build()?),
         );
         let stats_stage_cmplog = IfStage::new(
             |_, _, _, _| Ok(self.options.tui),
             tuple_list!(AflStatsStage::builder()
                 .map_observer(&edges_observer)
+                .map_name("map_feedback")
```

Comment:
Isn't this a case again where we should pass in a feedback and then get out the observer and feedback name, instead?

It's the exact same problem as in CalibrationStage, right?

Reply:
yeah, will fix